src/state/item/item-action-creators.js

```diff
@@ -92,7 +78,7 @@ export function saveMetadata() {
         itemApi.updateItemMetadata(activeItem, activeItemEditing)
             .then((success) => {
                 dispatch({ type: itemActionTypes.METADATA_SAVE_SUCCEEDED });
-                dispatch(fetchItems(currentPage));
+                dispatch(fetchItem(activeItem.id));
```

Comment:
i was doing this to update the grid on the left. so if you updated the title or author, it would update on the grid. i just pushed a change that will update only that specific item instead of fetching all of them